### PR TITLE
build/tools/amebasmart/bootloader: implement RDP verification feature

### PR DESCRIPTION
**Pull request by zhongnuo-tang**
1. add log for secure boot status, eg: "IMG1 SBOOT OFF/EN, IMG1 SBOOT SUCCESS"
2. add log for RDP status, eg: "RDP EN/OFF, RDP SUCCESS/FAIL"
3. implement feature to verify RDP, if RDP is encrypted with wrong key/iv, before-> module will be stuck during booting. now-> "RDP FAIL" will be printed.
4. please use secure_boot_script_amebasmart_v03_official to sign RDP/Secureboot.
5. increase hotfix wait time from 50ms to 100ms, please change BOOTLOADER_STRING='Autoboot in 100 milliseconds' in hotfix script
6. remove some unnecessary logs to prevent mix log

### CODE REVIEW

**Review by jylee9613 — APPROVED** [review]

File: build/tools/amebasmart/bootloader/target_loader.map [hunk 1]
```diff
@@ -117,297 +117,297 @@
...
+3000d660 t __FUNCTION__.1
+3000d670 t __FUNCTION__.2
+3000d68c t __FUNCTION__.3
+3000d748 t __FUNCTION__.1
+3000d7c0 t __FUNCTION__.0
+3000d7d0 T km4_bootloader_rev
+3000d808 D Boot_Agg_En
+3000d808 D __ram_image1_data_start__
+3000d809 D Boot_MemSwr_Only
+3000d80a D Boot_AP_Enbale
```
Comment (sunghan-chang) [inline build/tools/amebasmart/bootloader/target_loader.map:301]: Will you modify typo? Enable

Comment (sunghan-chang) [reply build/tools/amebasmart/bootloader/target_loader.map:301]: This is not request, just asking.

Comment (zhongnuo-tang) [reply build/tools/amebasmart/bootloader/target_loader.map:301]: Dear @sunghan-chang,
We will not modify this typo in this PR, we will modify it in the future PR for bootloader. Is that okay?

Comment (sunghan-chang) [reply build/tools/amebasmart/bootloader/target_loader.map:301]: Sure

**Review by sunghan-chang — APPROVED** [review]

**Review by jylee9613 — APPROVED** [review]

**Pull request merged**